en/04_Changelogs/5.0.0.md

```diff
@@ -23,12 +32,68 @@ title: 5.0.0 (unreleased)
   - [URLs generated by Silverstripe CMS don't include a trailing slash by default](#trailing-slash)
   - [CWP Agency Extensions](#cwp-agency-extensions)
 - [Features and enhancements](#features-and-enhancements)
-  - [Other features](#other-features)
+  - [Extension changes](#extension-changes)
+  - [Other new features](#other-features)
 - [Bugfixes](#bugfixes)
 
+## Change to supported modules {#supported-modules}
+
+Some Silverstripe CMS modules are commercially supported. Silverstripe commits to looking after those modules for the duration of the Silverstripe CMS 5 lifecycle.
+
+Review the list of [Commercially Supported Modules for Silverstripe CMS 5](/project_governance/supported_modules).
+
+### Community supported modules in Silverstripe CMS 5
```

Comment:
https://github.com/silverstripe/developer-docs/pull/147#discussion_r1099560123
Hmmmm I don't think this rename matches what the section is about. It's really about modules which used to be commercially supported. This heading makes me think it's going to talk about my random module which silverstripe doesn't know exists.

Please change this back - or otherwise keep it clearly on subject (e.g. "Previously commercially supported modules" or "Dropping commercial support for some modules").

I think the changes you've made in the _body_ of the text satisfy the discussion in https://github.com/silverstripe/developer-docs/pull/147#discussion_r1095165654